frontend/src/components/NavBarMenu.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
+import React, { useState } from "react";
+
+type MenuItem = {
+  label: string;
+  url: string;
+};
+
+const menuItems: MenuItem[] = [
+  {
+    label: "Books",
+    url: "/books",
+  },
+  {
+    label: "Characters",
+    url: "/characters",
+  },
+  {
+    label: "Movies",
+    url: "/movies",
+  },
+];
+
+const helpItems: MenuItem[] = [
+  {
+    label: "Docs",
+    url: "https://docs.potterdb.com/",
+  },
+  {
+    label: "Status",
+    url: "https://status.potterdb.com/",
+  },
+];
+
+const NavbarMenu = () => {
+  const [showMenu, setShowMenu] = useState<Boolean>(false);
+  return (
+    <>
+      <div
+        className={`w-screen h-screen bg-transparent fixed top-0 left-0 z-[20] ${
+          !showMenu ? "hidden" : ""
+        }`}
+        onClick={() => setShowMenu(false)}
+      />
+      <div className="flex flex-row">
+        <div className="flex-row items-center text-lg gap-3 mr-3 hidden sm:flex">
+          {helpItems.map((item) => {
+            return (
+              <a
+                key={item.label}
+                href={item.url}
+                target="_blank"
+                className="block rounded hover:bg-transparent hover:text-secondary text-white">
```

Comment:
Hover effects for the links should fit with the general link style, e.g. `text-gray-300 hover:text-white`. 
Well, it's not that it were a bad idea giving the links a purple color on hover, but I want to keep it in style with the footer links.